From a57f650e847f89aaeea3ecad857b39d15efba469 Mon Sep 17 00:00:00 2001
From: Mike Looijmans <mike.looijmans@topic.nl>
Date: Tue, 17 Jan 2017 08:52:37 +0100
Subject: [PATCH 03/10] topic_miami(plus) defconfig: Enable DFU RAM support

Allow sending firmware to RAM. Without this, the DFU support was not
of much use.

Signed-off-by: Mike Looijmans <mike.looijmans@topic.nl>
---
 configs/topic_miami_defconfig     | 1 +
 configs/topic_miamiplus_defconfig | 1 +
 2 files changed, 2 insertions(+)

diff --git a/configs/topic_miami_defconfig b/configs/topic_miami_defconfig
index 3f3052e..1261a5f 100644
--- a/configs/topic_miami_defconfig
+++ b/configs/topic_miami_defconfig
@@ -29,6 +29,7 @@ CONFIG_CMD_FAT=y
 CONFIG_CMD_FS_GENERIC=y
 CONFIG_OF_EMBED=y
 CONFIG_SPL_DM_SEQ_ALIAS=y
+CONFIG_DFU_RAM=y
 CONFIG_DM_GPIO=y
 CONFIG_DM_MMC=y
 CONFIG_ZYNQ_SDHCI=y
diff --git a/configs/topic_miamiplus_defconfig b/configs/topic_miamiplus_defconfig
index 19d758c..a551bb7 100644
--- a/configs/topic_miamiplus_defconfig
+++ b/configs/topic_miamiplus_defconfig
@@ -27,6 +27,7 @@ CONFIG_CMD_FS_GENERIC=y
 CONFIG_OF_EMBED=y
 CONFIG_OF_BOARD_SETUP=y
 CONFIG_SPL_DM_SEQ_ALIAS=y
+CONFIG_DFU_RAM=y
 CONFIG_DM_GPIO=y
 CONFIG_DM_MMC=y
 CONFIG_ZYNQ_SDHCI=y
-- 
1.9.1

